lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -22,6 +22,8 @@ const Runner = require('../../../lighthouse-core/runner');
 const Config = require('../../../lighthouse-core/config');
 const configJSON = require('../../../lighthouse-core/config/default.json');
 const log = require('../../../lighthouse-core/lib/log');
+const STORAGE_KEY = 'lighthouse_audits';
```

Comment:
Will we have any problems with versioning? Maybe we should drop the chrome extension version into the key as well.